tools/specgen/modeldefs.go

```diff
@@ -205,9 +205,8 @@ type ImagesCreateParameters struct {
 
 // ImagesListParameters for GET /images/json
 type ImagesListParameters struct {
-	MatchName string `rest:"query,filter"`
-	All       bool   `rest:"query"`
-	Filters   Args   `rest:"query"`
+	All     bool `rest:"query"`
+	Filters Args `rest:"query"`
```

Comment:
Looks like this now has `digests` too

Reply:
This struct mimics ImageListOptions https://github.com/moby/moby/blob/327daef5c19209898f11380a7bd02509d6675bcb/api/types/client.go#L239 

I've tried to specify value for digest=true or digest=false when quering docker api and it makes no difference. RepoDigests are shown by default

Reply:
Ok np